### PR TITLE
feat: US34193 [UI] View NFT DAO Details Page

### PR DESCRIPTION
**Pull request by anshulkhare-lab49**
Description:
feat: Created NFT DAO details page 

- Used DAODashboard to create NFTDAODashboard.
- Created DAOSettings component
- Added logo in dashboard header

Related issue(s):

Fixes #

- Moved common Member Interface out to DAO types from MultiSig.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by damiano-melcarne-lab49 — APPROVED** [review]

File: src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import { Outlet, useParams } from "react-router-dom";
-import { GovernanceDAODetails } from "@dex-ui/services";
+import { GovernanceDAODetails, Member } from "@dex-ui/services";
```
Comment (roshan-bisht-lab49) [inline src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx:2]:
```
import { GovernanceDAODetails, Member } from "@services";
```
Same here.

File: src/dex-ui/pages/dao/MembersList.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import { Flex, SimpleGrid, Text } from "@chakra-ui/react";
 import { Color, HashScanLink, HashscanData, Tag } from "@dex-ui-components";
 import * as R from "ramda";
-import { Member, MultiSigDAODetailsContext } from "./MultiSigDAODashboard/types";
+import { MultiSigDAODetailsContext } from "./MultiSigDAODashboard/types";
 import { useOutletContext } from "react-router-dom";
+import { Member } from "@dex-ui/services";
```
Comment (roshan-bisht-lab49) [inline src/dex-ui/pages/dao/MembersList.tsx:6]: Same as above

File: src/dex-ui/pages/dao/NFTDAODashboard/NFTDAODashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import { Outlet, useParams } from "react-router-dom";
+import { Member, NFTDAODetails } from "@dex-ui/services";
```
Comment (roshan-bisht-lab49) [inline src/dex-ui/pages/dao/NFTDAODashboard/NFTDAODashboard.tsx:2]: Same as above

**Review by roshan-bisht-lab49 — CHANGES_REQUESTED** [review]

File: src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx (hunk 1 above)
Comment (anshulkhare-lab49) [reply src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx:2]: Updated the Import.

File: src/dex-ui/pages/dao/NFTDAODashboard/NFTDAODashboard.tsx (hunk 3 above)
Comment (anshulkhare-lab49) [reply src/dex-ui/pages/dao/NFTDAODashboard/NFTDAODashboard.tsx:2]: Updated the import path.

File: src/dex-ui/pages/dao/MembersList.tsx (hunk 2 above)
Comment (anshulkhare-lab49) [reply src/dex-ui/pages/dao/MembersList.tsx:6]: Updated the import path.

**Review by roshan-bisht-lab49 — APPROVED** [review]

**Pull request merged**